lib/addons/p5.dom.js

```diff
@@ -550,12 +554,19 @@
    * }
    * </code></div>
    */
+
   p5.prototype.createSelect = function(mult) {
-    var elt = document.createElement('select');
-    if (mult){
-      elt.setAttribute('multiple', 'true');
+
+    if( typeof mult === 'object' && mult.elt.nodeName == 'SELECT' ){
+      var self = mult;
+      var elt = this.elt = mult.elt;
```

Comment:
I'm not quite sure what the intention is with this line, can you explain what's happening here?